src/components/checkout/address-form-header.js

```diff
@@ -1,35 +1,47 @@
 // Libraries
-import React from 'react'
+import React, { Component } from 'react'
```

Comment:
Had this as a `PureComponent` but this results in the component being rendered as `<address_form_header_AddressFormHeader>` instead of `<AddressFormHeader>`, which breaks specs in `shift-next`

Reply:
If I remember correctly, issues like this are due to the fact that we are checking for a presence of a component using a string, i.e. `'AddressFormHeader'`. This could be resolved by importing the address form header from shift-react-components in the spec and using the component class in the assertion instead.

Reply:
That's the current implementation. The snapshots have the correct/expected format - `<AddressFormHeader>`, however`<address_form_header_AddressFormHeader>` is rendered at runtime which does not match the snapshot. Note: this only occurs on CI builds.